packages/next-swc/crates/core/src/server_actions.rs

```diff
@@ -948,7 +948,9 @@ impl<C: Comments> VisitMut for ServerActions<C> {
                                     declare: false,
                                     decls: vec![VarDeclarator {
                                         span: DUMMY_SP,
-                                        name: Pat::Ident(ident.into()),
+                                        name: Pat::Ident(
+                                            Ident::new(export_name.clone().into(), DUMMY_SP).into(),
+                                        ),
```

Comment:
In a `export { A as B }` expression, `ident` will be `A` and `export_name` will be `B`.